### DISCUSSION TITLE
Disable Pre-Movement Chat Function?

### DISCUSSION THREAD

**Discussion by sean-perryman** [GriefPrevention/GriefPrevention#511] [Ideas]
My apologies if this is covered somewhere in the documentation, but is there a way to disable the pre-movement block? Looking at the original BigScary source it does not seem that this is the case.

**Comment by RoboMWM** [comment]
This only applies for new players. New players pre-1.13 were those who haven't yet achieved the Getting Wood achievement. You could add this to "forced-achievement" in spigot.yml to effectively disable this check. In the current version, since achievements are no longer a thing, GP directly checks the log pickup statistic.

I guess someone could PR yet another config option since this keeps getting asked

**Comment by RoboMWM** [comment]
This is probably a duplicate of a request somewhere in here though...
